### ISSUE TITLE
[Accessibility][Keyboard -Flow Chart]: Multiple node can't be selected by keyboard.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2031]
**User Experience** 
The keyboard dependent user will not be able to use the available functionality of select multiple nodes which can be accessed by mouse.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to the flow chart and try to select multiple nodes.
4. Observe the issue in the flow chart area.

**Actual:** 
With the help of keyboard, the user will not be able to select multiple nodes where as same can be done by mouse.

**Expected:**
The functionality which is accessible by mouse should be accessible by keyboard.

**MAS Impacted:** [MAS2.1.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2})

**Attachment:**
[MAS2.1.1_multiple selcetion is not allowed by keyboard.zip](https://github.com/microsoft/BotFramework-Composer/files/4230386/MAS2.1.1_multiple.selcetion.is.not.allowed.by.keyboard.zip)

**Comment by yeze322** [comment]
It's a feature, possible solutions:
1. Shift + ArrowKeys
2. Ctrl + A

**Comment by cwhitten** [comment]
Let's go with Shift + ArrowKeys to expand the selected region and map to the behavior of the mouse-based multi-selection today.

**Comment by ashish315** [comment]
As checked on the below environment, still we are not able to select multiple nodes with keyboard.

**Test Environment**
Browser: Microsoft Edge Canary Version: 85.0.531.1 (Official build)
OS build: 2004 (19635.1)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
Master folder downloaded on 05-06-2020

**Comment by Amit8527510735** [comment]
#Nonbenchmark;
